packages/clerk-js/src/ui/components/UserButton/UserButtonTrigger.tsx

```diff
@@ -13,24 +15,33 @@ export const UserButtonTrigger = withAvatarShimmer(
   forwardRef<HTMLButtonElement, UserButtonTriggerProps>((props, ref) => {
     const { sx, ...rest } = props;
     const { user } = useUser();
+    const { showName } = useUserButtonContext();
 
     return (
       <Button
         elementDescriptor={descriptors.userButtonTrigger}
         variant='roundWrapper'
-        sx={[theme => ({ borderRadius: theme.radii.$circle }), sx]}
+        sx={[theme => ({ borderRadius: showName ? theme.radii.$xl : theme.radii.$circle }), sx]}
```

Comment:
@marcelscruz  Should we go with a smaller radius here? Do be similar with the OrganizationSwitcher one!